content/cloud-docs/api-docs/configuration-versions.mdx

```diff
@@ -346,6 +352,40 @@ curl \
   https://archivist.terraform.io/v1/object/4c44d964-eba7-4dd5-ad29-1ece7b99e8da
 ```
 
+## Archive a Configuration Version
+
+`POST /configuration-versions/:configuration_version_id/actions/archive`
+
+| Parameter                   | Description                                    |
+| --------------------------- | ---------------------------------------------- |
+| `configuration_version_id`  | The ID of the configuration version to archive |
+
+Notifies Terraform Cloud to discard the uploaded `.tar.gz` file associated with this configuration version.
+
+Only configuration versions created via the API or CLI, in an `uploaded` state, and with no associated runs in progress can be archived using this endpoint. Otherwise, calling this endpoint will result in an error.
+
+VCS configuration versions are archived automatically once their runs are complete, then re-fetched for subsequent runs.
```

Comment:
Just trying to make it in active voice, which is easier to read :-) 

```suggestion
This endpoint notifies Terraform Cloud to discard the uploaded `.tar.gz` file associated with this configuration version. This endpoint can only archive configuration versions that were created with the API or CLI, are in an `uploaded` [state](#configuration-version-states), and have no runs in progress. Otherwise, calling this endpoint will result in an error. 

Terraform Cloud automatically archives configuration versions created through VCS when associated runs are complete and then re-fetches the files for subsequent runs.
```